src/vsphere_cpi/Gemfile

```diff
@@ -7,6 +7,8 @@ gem 'builder',     '~>3.2.3'
 gem 'nokogiri',    '~>1.8.1'
 gem 'mono_logger', '~>1.1.0'
 gem 'oga', '2.3'
+gem 'typhoeus', '~> 1.0', '>= 1.0.1'
```

Comment:
Do we have to use the `typhoeous` client? We've already got `httpclient`.

Reply:
This is for nsxt generated client. it uses typhoeus